wm/property.go

```diff
@@ -22,6 +23,44 @@ func windowName(x *xgbutil.XUtil, win xproto.Window) string {
 	return name
 }
 
+func windowClass(x *xgbutil.XUtil, win xproto.Window) []string {
+	class, err := xprop.PropValStrs(xprop.GetProperty(x, win, "WM_CLASS"))
+	if err != nil {
+		class, err := xprop.PropValStrs(xprop.GetProperty(x, win, "_NET_WM_CLASS"))
+		if err != nil {
+			return []string{"Noclass"}
```

Comment:
Surely in these 3 cases a zero value ("") would be better than an arbitrary string?
If this string is needed then it should probably be a public const so that others can compare to it...